structs.go

```diff
@@ -1,37 +1,36 @@
 package rcmd
 
-// CmdResult stores information about the executed cmd
+// CmdResult stores information about the executed cmd.
 type CmdResult struct {
 	Stdout   string `json:"stdout,omitempty"`
 	Stderr   string `json:"stderr,omitempty"`
 	ExitCode int    `json:"exit_code,omitempty"`
 }
 
-// ExecSettings controls settings related to R execution
+// ExecSettings controls settings related to R execution.
 type ExecSettings struct {
 	WorkDir string `json:"work_dir,omitempty"`
 }
 
-// RVersion contains information about the R version
+// RVersion contains information about the R version.
 type RVersion struct {
 	Major int
 	Minor int
 	Patch int
 }
 
-// RSettings controls settings related to managing libraries
-// if AsUser is set, R will be run as the user would launch from their normal session
-// with no interception/injection of library paths or environment variables for R_LIBS_SITE and R_LIBS_USER
+// RSettings controls settings related to managing libraries.
+// if AsUser is set, R will be run as the user would launch from their normal session.
```

Comment:
Not sure if it's taboo, but I feel like the "i" in "if" should be capitalized since it's a new phrase as enforced by godot. I also don't feel like line 24 is a phrase that could stand on its own with a "." to terminate. Personal thing, not an issue and it's probably just because we're trying to placate godot.